src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Program.cs

```diff
@@ -43,6 +43,14 @@
 #endif
 app.UseStaticFiles();
 
+#if (UseServer && UseWebAssembly)
+app.MapRazorComponents<App>().AddServerRenderMode().AddWebAssemblyRenderMode();
```

Comment:
I think this might be more readable on separate lines.

```suggestion
app.MapRazorComponents<App>()
    .AddServerRenderMode()
    .AddWebAssemblyRenderMode();
```

Reply:
OK, will change it